components/apim-analytics/org.wso2.analytics.solutions.apim.analytics/components/org.wso2.sp.extension.siddhi.io.mgwfile/src/main/java/org/wso2/extension/siddhi/io/mgwfile/task/MGWFileCleanUpTask.java

```diff
@@ -55,10 +55,15 @@ private Date getLastKeptDate(int fileRetentionDays) {
 
     @Override
     public void run() {
-        log.debug("UploadUsage Cleaning task started");
+        if (log.isDebugEnabled()) {
```

Comment:
Since we use SL4J, no need to check `isDebugEnabled` as it is automatically checked

Reply:
Refer https://stackoverflow.com/q/17850614/1577286